### ISSUE TITLE
[Bug]: Pupil Labs Eyetracker Plugin fails to load

### ISSUE THREAD

**Issue by domstoppable** [psychopy/psychopy#5880]
### PsychoPy Version

2023.2.1

### What OS are your PsychoPy running on?

Windows 10, Linux-based systems

### Bug Description

The Pupil Labs plugin (perhaps others?) fails to load when running an experiment. It looks as though iohub is looking for yaml files in the old (internal) location, rather than in the plugin's install directory.

```bash
Error during device creation ....
Traceback (most recent call last):
  File "/home/dom/pupil/projects/neon-psychopy/venv/lib/python3.9/site-packages/psychopy/iohub/server.py", line 715, in createNewMonitoredDevice
    dev_data = self.addDeviceToMonitor(dev_cls_name, dev_conf)
  File "/home/dom/pupil/projects/neon-psychopy/venv/lib/python3.9/site-packages/psychopy/iohub/server.py", line 847, in addDeviceToMonitor
    _dconf = yload(open(dev_conf_pth, 'r'), Loader=yLoader)
FileNotFoundError: [Errno 2] No such file or directory: '/home/dom/pupil/projects/neon-psychopy/venv/lib/python3.9/site-packages/psychopy/iohub/devices/eyetracker/hw/pupil_labs/pupil_core/default_eyetracker.yaml'
Error during device creation ....
Traceback (most recent call last):
  File "/home/dom/pupil/projects/neon-psychopy/venv/lib/python3.9/site-packages/psychopy/iohub/server.py", line 715, in createNewMonitoredDevice
    dev_data = self.addDeviceToMonitor(dev_cls_name, dev_conf)
  File "/home/dom/pupil/projects/neon-psychopy/venv/lib/python3.9/site-packages/psychopy/iohub/server.py", line 847, in addDeviceToMonitor
    _dconf = yload(open(dev_conf_pth, 'r'), Loader=yLoader)
FileNotFoundError: [Errno 2] No such file or directory: '/home/dom/pupil/projects/neon-psychopy/venv/lib/python3.9/site-packages/psychopy/iohub/devices/eyetracker/hw/pupil_labs/pupil_core/default_eyetracker.yaml'.

The stacktrace below is from a pip install on Debian Linux, but the error is also observed on Windows using the standalone installer. 

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/dom/pupil/projects/neon-psychopy/venv/lib/python3.9/site-packages/psychopy/iohub/server.py", line 649, in _addDevices
    self.createNewMonitoredDevice(dev_cls_name, dev_conf)
  File "/home/dom/pupil/projects/neon-psychopy/venv/lib/python3.9/site-packages/psychopy/iohub/server.py", line 728, in createNewMonitoredDevice
    raise ioHubError('Error during device creation ....')
psychopy.iohub.errors.ioHubError: ioHubError:
Args: ('Error during device creation ....',)

GET_DEV_INTERFACE_ERROR: _getRPCInterface returned: None
Traceback (most recent call last):
0.7681     WARNING     Monitor specification not found. Creating a temporary one...
ioHub Server Process Completed With Code:  0
  File "/home/dom/pupil/projects/neon-psychopy/exps/test_lastrun.py", line 494, in <module>
    run(
  File "/home/dom/pupil/projects/neon-psychopy/exps/test_lastrun.py", line 406, in run
    endExperiment(thisExp, win=win, inputs=inputs)
  File "/home/dom/pupil/projects/neon-psychopy/exps/test_lastrun.py", line 452, in endExperiment
    inputs['eyetracker'].setConnectionState(False)
  File "/home/dom/pupil/projects/neon-psychopy/venv/lib/python3.9/site-packages/psychopy/iohub/client/__init__.py", line 153, in __getattr__
    raise AttributeError(self, name)
AttributeError: (<psychopy.iohub.client.ioHubDeviceView object at 0x7fc8d6ddcc40>, 'setConnectionState')
################ Experiment ended with exit code 1 [pid:30731] #################```

### Expected Behaviour

The Pupil Labs plugin should load and work as it did before the migration to plugins.

### Steps to Reproduce

1. Uninstall PsychoPy and delete `~/.psychopy3/packages/psychopy-eyetracker-pupil-labs` if present
2. Install PsychoPy
3. Launch PsychoPy and use the plugin manager to install the Pupil Labs plugin
4. Configure or open an experiment with the eyetracker hardware configured for Pupil Labs
5. Run the experiment



### Additional context

_No response_

**Comment by TEParsons** [comment]
Currently have a possible fix in the works - I'll tag this issue in the pull request when I submit :)

**Comment by peircej** [comment]
Now updated fix by #5887
